### ISSUE TITLE
[Quest][Classic] Scarlet Diversions/[Item]Scourge Banner

### ISSUE THREAD

**Issue by ImprezaWoW** [dalaranwow/dalaran-wow#3883]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (OLD - ... tags should no longer be used)
[//]: # (Priority - ... tags are assigned by team members)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Description**:
For the quest, once you accept it, you need to get the Flame in a Bottle from next to the quest giver, go to the camp at around Scarlet Crusade Forward Camp 40, 53 , kill the mobs around the area (not necessarily, but it definitely helps if you're low level and don't want to die), burn the tent and place a Scourge Banner.

First tickets about it rolled in today, so by any chance it might be related to any of the other recently applied fixes, maybe not, but it used to work.

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Once the tent is burned, you can not place the banner. In the whole area including around the tent it says "Requires Scarlet Crusade Forward Camp"
**Expected behaviour**: Tell us what should happen instead
The banner should be placable in the area.
**Steps to reproduce the problem**:

1. Accept the quest;
2. Kill NPCs and/or burn the tent;
3. Try placing the flag.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
[Scarlet Diversions](http://db.vanillagaming.org/?quest=5096) quest ID 5096
[Scourge Banner](http://db.vanillagaming.org/?item=12807) item ID 12807

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
DB/Gameobject: set positive spawntimesecs for 2 quest objects

* the spellfocusobject had a negative spawntime, which caused this issue. Fixed on ptr
* commit: 88a15486638d91c76637eec5bdf641d7e07a6aba

The issue is fixed on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by Rushor** [comment]
i'm not responsible for the time where the updates get merged on live. i only produce the fixes.

**Comment by Luwyola** [comment]
Please fix that Quest, still doesn't work. Cannot place the flag.

**Comment by Rushor** [comment]
The issue is fixed on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by TeridanDW** [comment]
Few minor issues here:
 - when a player use item http://wotlk.openwow.com/item=12807 the flag is planted exacly on the same cordinates as the player, it should be around him.
 - when a player use item http://wotlk.openwow.com/item=12807 he gets double quest credit message
 - player can skip burning the tend (only the flag gives the credit), however in the quest log it says both, not sure if it's bug

reference from vanilla retail from 2006:
https://www.youtube.com/watch?v=_9jV2sj97m4

**Comment by Rushor** [comment]
> when a player use item http://wotlk.openwow.com/item=12807 he gets double quest credit message

that's actually an issue for all quests at the moment

**Comment by Rushor** [comment]
> player can skip burning the tend (only the flag gives the credit), however in the quest log it says both, not sure if it's bug

it's a bug yes, you should only be able to summon the flag if the tend was burned down

**Comment by TeridanDW** [comment]
That's not the case, i have no issues summoning the flag with the tend been up.

**Comment by Rushor** [comment]
kk will review this

**Comment by Rushor** [comment]
Scripts/EasternKingdoms: Quest - Scarlet Diversions - Remove static spawned spellfocus object

* only sumon spellfocus object once the tend was burned down, then you are able to put up the flag 
* spawn flag a bit away from the caster
* updates: https://github.com/dalaranwow/dalaran-wow/issues/3883
* commit: 5bb89afeb95996f800febfc9d47c9df3bbec6aae

![wow_2017-03-30_15-04-48](https://cloud.githubusercontent.com/assets/5568820/24505657/ff0754ba-155a-11e7-8f58-19569367ff17.png)
![wow_2017-03-30_15-05-45](https://cloud.githubusercontent.com/assets/5568820/24505658/ff0d6206-155a-11e7-8fdf-5c2b754e0d91.png)
![wow_2017-03-30_15-05-53](https://cloud.githubusercontent.com/assets/5568820/24505659/ff14eef4-155a-11e7-99f5-e329db031c65.png)
![wow_2017-03-30_15-06-01](https://cloud.githubusercontent.com/assets/5568820/24505660/ff1b25c6-155a-11e7-9b12-c9a382f714ed.png)


The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by Rushor** [comment]
> when a player use item http://wotlk.openwow.com/item=12807 he gets double quest credit message

was fixed by tc in the last year, so we only need to wait for further tc merges

**Comment by Virmire** [comment]
I don't know which thread to post this, but this quest apears to be broken again. Tried on 28th June, encountered the problem mentioned in the first post: flag can not be planted, despite burning the tent down.

**Comment by notbetatester** [comment]
Still bug i just run into this quest today. the box next to the npc that give you Flame in a Bottle is not working, i play horde so idk about alliance.

**Comment by Lothloryen** [comment]
Hello @notbetatester, 

I have just checked it and it worked just fine, maybe it was something related to your character. Try it again, please. If it does not work I will take a further look.

Thank you for the update.
